core/src/main/scala/org/bitcoins/core/psbt/PSBT.scala

```diff
@@ -907,6 +996,15 @@ case class InputPSBTMap(elements: Vector[InputPSBTRecord]) extends PSBTMap {
     InputPSBTMap((this.elements ++ other.elements).distinct)
   }
 
+  def toUTXOSpendingInfo(
```

Comment:
I don't understand the point of this function, why would you want a spending info after it is signed?

Reply:
This is for the case where someone is doing something interactive and given some finalized `InputMap`, wants to do something else (non-interactive) whose interface expects `UTXOSpendingInfo`s instead of `PSBT` things (since `UTXOSpendingInfo` is the canonical signing information thing).

I guess one could argue that they could just use `TransactionInput` somehow and also that this case doesn't really exist yet (I originally thought I would use this in DLC stuff but now it seems I will be using PSBTs internally), but since the work is done and it isn't too complicated I figured I'd leave it here and we can get rid of it if it becomes an annoyance.

@Christewart thoughts?